viewer/package.json

```diff
@@ -34,6 +34,7 @@
     "react-router-dom": "^6.21.0",
     "react-scripts": "5.0.1",
     "react-select": "^5.8.0",
+    "styled-components": "^6.1.12",
```

Comment:
This did look used.

Reply:
Oh, yup. I was maybe going to use it, but went with the tailwind approach instead.